tests/Functional/Driver/PgSQL/BooleanBindingTest.php

```diff
@@ -0,0 +1,63 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Doctrine\DBAL\Tests\Functional\Driver\PgSQL;
+
+use Doctrine\DBAL\ParameterType;
+use Doctrine\DBAL\Schema\Table;
+use Doctrine\DBAL\Tests\FunctionalTestCase;
+use Doctrine\DBAL\Tests\TestUtil;
+
+class BooleanBindingTest extends FunctionalTestCase
+{
+    protected function setUp(): void
+    {
+        parent::setUp();
```

Comment:
There's no need to call setup/teardown on the parent class. They are known to be no-op.